### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by christophergreennov**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,8 +1,5 @@
 ﻿Working with code owners
 https://docs.github.com/en/repositories/managing-your-repositorys-settings-and-features/customizing-your-repository/about-code-owners
 
-Rebasing
-https://docs.github.com/en/get-started/using-git/about-git-rebase
```
Comment (frank-laskowski) [inline README.md]: this shouldn't have been removed.

**Review by frank-laskowski — CHANGES_REQUESTED** [review]
Please check it

File: README.md (hunk 1 above)
Comment (christophergreennov) [reply README.md]: I removed it by mistake.  Thanks Ill put it back.

**Review by frank-laskowski — APPROVED** [review]
This was approved.

**Pull request merged**